HesoyamHospital/WebApplication/Appointments/AppointmentController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using WebApplication.Appointments.Service;
+
+namespace WebApplication.Appointments
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AppointmentController : ControllerBase
+    {
+        private readonly IAppointmentService _appointmentService;
+        private readonly long defaultPatientId = 500;
```

Comment:
I suppose this defaultPatientId is reserved to represent non-registered or non-existing patients. In that case I would put it as a global constant and of course take care of preventing registration of new patients with this id.

Reply:
It represents a logged in patient since that functionality is not implemeted yet. Double registration of a patient with the same id value is not possible since it already exists in the database, otherwise I wouldn't be able to test how the new feature for a logged in user works.